api-reference/10 UI Components/dxButtonGroup/1 Configuration/selectionMode.md

```diff
@@ -6,6 +6,31 @@ default: 'single'
 ---
 ---
 ##### shortDescription
-Specifies whether a single or multiple buttons can be in the selected state simultaneously.
+Specifies a button selection mode.
```

Comment:
```suggestion
Specifies the button selection mode.
```

Reply:
Although you can specify any button selection mode, you are specifyinh a specific button selection mode